### PR TITLE
Release wakelock_macos

### PR DESCRIPTION
**Pull request by creativecreatorormaybenot**
## Description

"Releases" `wakelock_macos` by including it in the `wakelock` package and releasing a new version of that.

This finishes up #86 & #87.

cc @wilsonowilson

### CODE REVIEW

**Comment by creativecreatorormaybenot** [comment]
@wilsonowilson I tested the current head of this branch on web, macOS, and iOS and everything seems to be working perfectly 👌 

I think that the checks should also go all green now (which they actually did not before your PR haha).  
This means that the macOS implementation is now integration tested on macOS 🔥  (and the iOS integration tests run again as well).

**Comment by wilsonowilson** [comment]
That's fantastic 🚀

Just a quick thought. Shouldn't the plugin description in the pubspec.yaml also reference macOS as a supported platform now?

**Comment by creativecreatorormaybenot** [comment]
@wilsonowilson You are totally right; thanks for pointing it out. I forgot about that place 🙈 

I only changed it in the various READMEs.

**Comment by creativecreatorormaybenot** [comment]
@wilsonowilson Reminds me of https://github.com/flutter/flutter/pull/70968, where they also forgot to update that one README haha.

**Comment by creativecreatorormaybenot** [comment]
I think that I am done now :)

When the checks give green light and you do as well @wilsonowilson, this can be merged 🚀 Cannot thank you enough!

**Comment by wilsonowilson** [comment]
Thanks too! It was really fun working on this! And from what I've seen so far, everything looks great 🎉

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -105,4 +107,5 @@ jobs:
         if: matrix.package == 'wakelock_macos' && matrix.channel == 'dev'
         run: |
           cd wakelock/example
+          flutter config --enable-macos
```
Comment (wilsonowilson) [inline .github/workflows/checks.yml:110]: Shouldn't this be --enable-macos-desktop?

**Review by wilsonowilson — COMMENTED** [review]
Fix config

File: .github/workflows/checks.yml (hunk 1 above)
Comment (creativecreatorormaybenot) [reply .github/workflows/checks.yml:110]: @wilsonowilson Yep! :) Thanks for pointing it out. Luckily, the checks would have also caught it.

I also realized that I have to fix something else related to testing. See my next commit.

**Pull request merged**